lib/common/constants.js

```diff
@@ -176,7 +176,7 @@ function constantsFactory (path) {
         Regex: {
             Base64: /^(?:[A-Z0-9+\/]{4})*(?:[A-Z0-9+\/]{2}==|[A-Z0-9+\/]{3}=|[A-Z0-9+\/]{4})$/i,
             Encrypted: /^(?:[A-Z0-9+\/]{4})*(?:[A-Z0-9+\/]{2}==|[A-Z0-9+\/]{3}=|[A-Z0-9+\/]{4})\.(?:[A-Z0-9+\/]{4})*(?:[A-Z0-9+\/]{2}==|[A-Z0-9+\/]{3}=|[A-Z0-9+\/]{4})$/i, // jshint ignore:line
-            MacAddress: /^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})$/,
+            MacAddress: /^([0-9a-fA-F]{2}[:-]){5}([0-9a-fA-F]{2})$|^([0-9a-fA-F]{4}[.]){2}([0-9a-fA-F]{4})$/,
```

Comment:
@scott-xu:
Many people like to use vertical screen to program, meanwhile, we need to view our code in many devices, even in some small screen devices, like iPad or Phone, of course, much more other reason.

Current, we limit the line character size to 100. We may can extend to 128, and don't want to be more high. @RackHD/corecommitters 

![ ](http://mobilitydigest.com/wp-content/uploads/2013/09/go-vertical.jpg)